app/views/users/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= stylesheet_link_tag('index') %>
+<div class="container">
+  <div class="user">
+   <img src="<%= image_path('example.com.png') %>" alt="image">
+    <div class= "text-container">
+      <h1>Username</h1>
+      <p class= "inner-text">Number of posts x</p>
+    </div>
+  </div>
+  <div>
+    <h2>Bio</h2>
+    <p>post 1</p>
+    <p>post 2</p>
+    <p>post 3</p>
+  </div>
+</div>
```

Comment:
- You have done well setting up your view file for the `user#show action`, but according to the project requirements, you are to set up a `user` controller with the show action, please fix this 😄 
**OPTIONAL** - Also, please note that you are not required to have any logic in your view files for this milestone, your view file is allowed to have just a placeholder. Please fix this 👍 👍